### ISSUE TITLE
Setting a style on an ExcelRow or ExcelRangeRow resets unrelated style objects in an erratic way

### ISSUE THREAD

**Issue by xeniorn** [EPPlusSoftware/EPPlus#1096]
### EPPlus usage

Non-Commercial use 

### Environment

Windows 10

### Epplus version

6.2.10

### Spreadsheet application

Excel

### Description

We have a report generating system that includes these steps:
1) set global font on the entire sheet
2) set a specific font on some columns
3) set background formatting on some rows

Upon upgrade from EPPlus v5 to v6, we started observing wrong fonts appearing in the sheet. I was able to isolate the behavior in a minimal example:

1) set global font to Tahoma
2) set column 2 font to another font
3) set row 2 fill pattern type

Fonts before step 3 are as expected, fonts after set 3 change in an unusual way:
a) in row 2 columns from 4 onwards are changed to Calibri 11 (default font)
b) columns 1-3 are fine
note: same happens if an ExcelRangeRow object is used instead to apply the style (Cells[2,1].EntireRow instead of Row(2))

code:

```
static void Main(string[] args)
{
    ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
    var package = new ExcelPackage();
    var workbook = package.Workbook;


    var wss = workbook.Worksheets.Add("test");

    var cells = new List<ExcelRange>()
    {
        wss.Cells[1, 1],
        wss.Cells[2, 1],
        wss.Cells[3, 1],
    };

    wss.Cells.Style.Font.Name = "Tahoma";
    wss.Cells.Style.Font.Size = 10;

    cells.ForEach(x =>
    {
        x.Value = "test";
        //x.Style.Font.Name = "Tahoma";
        //x.Style.Font.Size = 10;
    });

    cells = new List<ExcelRange>()
    {
        wss.Cells[1, 2],
        wss.Cells[2, 2],
        wss.Cells[3, 2],
    };

    cells.ForEach(x =>
    {
        x.Value = "test";
        //x.Style.Font.Size = 10;
    });
    
    wss.Column(2).Style.Font.Name = "Wingdings";

    cells = new List<ExcelRange>()
    {
        wss.Cells[1, 3],
        wss.Cells[2, 3],
        wss.Cells[3, 3],
        wss.Cells[1, 4],
        wss.Cells[2, 4],
        wss.Cells[3, 4],
        wss.Cells[1, 5],
        wss.Cells[2, 5],
        wss.Cells[3, 5],
    };

    cells.ForEach(x =>
    {
        x.Value = "hola";
        //x.Style.Font.Name = "Wingdings";
        //x.Style.Font.Size = 10;
    });


    void DebugGetFontInfo()
    {
        for (var row = 1; row <= 3; row++)
        {
            for (var column = 1; column <= 5; column++)
            {
                var range = wss.Cells[row, column];
                Console.Write(range.Style.Font.Name + range.Style.Font.Size + ",");
            }
            Console.WriteLine();
        }
    }


    Console.WriteLine("before:");
    DebugGetFontInfo();

    var x = wss.Row(2);
    x.Style.Fill.PatternType = ExcelFillStyle.Solid;
    //x.Style.Fill.BackgroundColor.SetColor(Color.Pink);

    Console.WriteLine("after:");
    DebugGetFontInfo();
}
```

Output:
```
before:
Tahoma10,Wingdings10,Tahoma10,Tahoma10,Tahoma10,
Tahoma10,Wingdings10,Tahoma10,Tahoma10,Tahoma10,
Tahoma10,Wingdings10,Tahoma10,Tahoma10,Tahoma10,
after:
Tahoma10,Wingdings10,Tahoma10,Tahoma10,Tahoma10,
Tahoma10,Wingdings10,Tahoma10,Calibri11,Calibri11,
Tahoma10,Wingdings10,Tahoma10,Tahoma10,Tahoma10,
```

**Comment by JanKallman** [comment]
Thanks, this is a bug, I'll try to provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.2.11 and 7.0.0
